<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawEcharts</class>
 <widget class="QWidget" name="DrawEcharts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,6,3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWebEngineView" name="WebEngineView_Histogram1" native="true"/>
     </item>
     <item>
      <widget class="QWebEngineView" name="WebEngineView_LinBar1" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWebEngineView" name="WebEngineView_radar1" native="true"/>
     </item>
     <item>
      <widget class="QWebEngineView" name="WebEngineView_radar2" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWebEngineView" name="WebEngineView_Histogram2" native="true"/>
     </item>
     <item>
      <widget class="QWebEngineView" name="WebEngineView_LinBar2" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
